greengrass-ipc-model/main.smithy

```diff
@@ -867,7 +867,9 @@ structure GetSecretValueRequest {
     /// (Optional) The ID of the version to get. If you don't specify versionId or versionStage, this operation defaults to the version with the AWSCURRENT label.
     versionId: String,
     /// (Optional) The staging label of the version to get. If you don't specify versionId or versionStage, this operation defaults to the version with the AWSCURRENT label.
-    versionStage: String
+    versionStage: String,
+    /// (Optional) Set to True to fetch latest secret from the cloud when this API is invoked.
+    refresh: Boolean
```

Comment:
what is the default?

Reply:
Document the default value.

Reply:
If nothing is set in the request, it is considered as `null`. Verified that from the generated files.

Reply:
Document what the default will be when nothing is provided. What is the behavior? Don't say it is null; say what null means.

Reply:
What `null` means depends on the handler implementation. Since we take it as `false`, I'm documenting the same.